docs/reference/supported-environments.md

```diff
@@ -33,7 +39,42 @@ For example, 1.2+ means support for the minor version 2, and any higher minors (
 
 ## Camunda 8 Self-Managed
 
-We highly recommend running Camunda 8 Self-Managed in a Kubernetes environment. We provide officially supported [Helm Charts](/self-managed/platform-deployment/helm-kubernetes/overview.md) for this. Please follow the [Installation Guide](/self-managed/platform-deployment/overview.md) to learn more about installation possibilities.
+### Deployment options
+
+With the right configuration, Camunda 8 Self-Managed can be deployed on any [Certified Kubernetes](https://www.cncf.io/training/certification/software-conformance/#benefits) distribution (cloud or on-premise). However, we officially test and support a specific list of platforms.
+
+The following are tested and supported deployment options for Kubernetes, Docker, and manual installation:
+
+- [Stock Kubernetes](/self-managed/platform-deployment/helm-kubernetes/overview.md)
```

Comment:
We need to clarify what does it mean -Stock Kubernetes. It is not a standard term that used in CNCF. Does it mean upstream or certified?
imo we should support any certified distribution and specifically highlight tested platforms. (In a similar fashion to Elastic or Confluence)

Reply:
This is a great callout. I think this should likely just be referenced as Kubernetes (k8s).

We can add style guide assistance on this once agreed upon here.

Reply:
ok. Let's keep it as is, so we can progress here, and I will clarify and make another Epic to clarify this specific point and we will make changes throughout the docs.